### PR TITLE
Modify the gradle build script to use artifactory for the athena JDBC driver

### PR DESCRIPTION
**Pull request by favilo**
We will have to get the other dependencies later

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -50,18 +62,20 @@ dependencies {
     implementation group: 'mysql', name: 'mysql-connector-java', version: '8.0.9-rc'
     implementation group: 'org.postgresql', name: 'postgresql', version: '42.2.1'
     implementation group: 'net.snowflake', name: 'snowflake-jdbc', version: '3.6.25'
-    implementation group: 'com.syncron.amazonaws', name: 'simba-athena-jdbc-driver', version: '2.0.2'
+    implementation group: 'com.syncron.amazonaws', name: 'simba-athena-jdbc-driver', version: '2.0.25'
```
Comment (favilo) [inline build.gradle:65]: This is not the filename that came with the zip file, I don't know how to change the base filename that gradle searches for.

Comment (louiszuckerman) [reply build.gradle:65]: should that be version `2.0.5` instead of `2.0.25`?

Comment (favilo) [reply build.gradle:65]: Nope, I think you misread it originally, this was the version from aws that you sent me

Comment (louiszuckerman) [reply build.gradle:65]: Ahahaha yes you're right, I totally misread that. Wow we were far behind!

**Comment by favilo** [comment]
Alright, @louiszuckerman , think I can hit the button on this? Or should I wait for someone else with more domain knowledge?

**Review by JRemitz — APPROVED** [review]

**Pull request merged**